benchmarks/spack/cosma8/compute-node/spack.yaml

```
@@ -593,4 +372,4 @@ spack:
       - spec: xz@5.2.5
         prefix: /cosma/local/intel/oneAPI_2021.1.8/intelpython/latest
       - spec: xz@5.2.2
-        prefix: /usr
+        prefix: /usr
```

Missing newline (which was there before this change):
```suggestion
        prefix: /usr

```

---

Now,it is okay I guess.